Marlin/Marlin_main.cpp

```diff
@@ -1579,7 +1647,7 @@ static void setup_for_endstop_move() {
       feedrate = homing_feedrate[Z_AXIS];
 
       // Move down until the Z probe (or endstop?) is triggered
-      float zPosition = -(Z_MAX_LENGTH + 10);
+      float zPosition = MIN_PROBE_PT;
```

Comment:
This should not be altered. The length given is meant to be excessive, so that even if the Z probe is very far away from the bed it will still reach it. We can't use a "point" here because the Z coordinate is not really "known" when probing.